### PR TITLE
[TrafficManager] Real User Metrics (RUM) fixes

### PR DESCRIPTION
**Pull request by allencal**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json [hunk 1]
```diff
@@ -430,30 +427,32 @@
...
-       },
-       "allOf": [
-         {
-           "$ref": "#/definitions/ProxyResource"
-         }
+    "TrafficManagerUserMetricsKeyModel": {
+      "properties": {
+        "id": {
+          "type" : "string",
+          "description": "Fully qualified resource Id for the resource. Ex - /providers/Microsoft.Network/trafficManagerUserMetricsKeys/default"
```
Comment (lmazuel) [inline specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json]: It seems you removed "default" evrywhere else, is the description still accurate?

File: specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json [hunk 2]
```diff
@@ -126,14 +126,11 @@
         ],
         "responses": {
           "200": {
-            "description": "The subscription-level Traffic Manager Realtime User Metrics key.",
+            "description": "The subscription-level Traffic Manager Real User Metrics key.",
             "schema": {
               "$ref": "#/definitions/TrafficManagerUserMetricsKeyModel" 
             }
           },
-          "204": {
```
Comment (lmazuel) [inline specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json]: Does this mean that 204 is impossible, or you want now a 204 to raise an exception?

Comment (allencal) [reply specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json]: The "Get" operation never returns a 204. If the subscription has no RUM key, it returns a 200 and the body returns a blank key ("key": "")

File: specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json (hunk 1 above)
Comment (allencal) [reply specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json]: You're right, I think I should change this here. The "/default" is still returned in this description, though it shouldn't be; the call with "/default" on the end fails. I'll remove it here, and also in the API operation.

**Review by lmazuel — APPROVED** [review]
LGTM. Tell me if you sign-off or if you have more to add.

**Comment by allencal** [comment]
I do not have more to add. I'm not authorized to merge this pull request. What happens next?

**Comment by lmazuel** [comment]
@allencal Next is I merge :)
If your question is "next-next" like SDK/CLI, please send me an email (my github alias is my MS alias)

**Pull request merged**